node/clients/checkout.ts

```diff
@@ -11,7 +11,7 @@ const CHECKOUT_ENDPOINT = 'api/checkout/pvt/configuration/orderForm'
 
 export const CHECKOUT_APP = 'b2b-quotes-graphql'
 
-export class Checkout extends JanusClient {
+export default class Checkout extends JanusClient {
```

Comment:
Why did you change to default export? Just understanding your changes

Reply:
Mostly to be consistent with the other clients. It affects how they're imported in `node/clients/index.ts`